src/components/tile-details-panel/texture-section.tsx

```diff
@@ -0,0 +1,418 @@
+  }
+  const supportedFormats = getSupportedGPUTextureFormats(gl);
+
+  switch (format) {
+    case COMPRESSED_RGB_S3TC_DXT1_EXT:
+    case COMPRESSED_RGBA_S3TC_DXT3_EXT:
+    case COMPRESSED_RGBA_S3TC_DXT5_EXT:
+    case COMPRESSED_RGBA_S3TC_DXT1_EXT:
+      return supportedFormats.has("dxt");
+
+    case COMPRESSED_RGB_PVRTC_4BPPV1_IMG:
+    case COMPRESSED_RGBA_PVRTC_4BPPV1_IMG:
+    case COMPRESSED_RGB_PVRTC_2BPPV1_IMG:
+    case COMPRESSED_RGBA_PVRTC_2BPPV1_IMG:
+      return supportedFormats.has("pvrtc");
+
+    case COMPRESSED_RGB_ATC_WEBGL:
+    case COMPRESSED_RGBA_ATC_EXPLICIT_ALPHA_WEBGL:
+    case COMPRESSED_RGBA_ATC_INTERPOLATED_ALPHA_WEBGL:
+      return supportedFormats.has("atc");
+
+    case COMPRESSED_RGB_ETC1_WEBGL:
+      return supportedFormats.has("etc1");
+
+    case COMPRESSED_RGBA_ASTC_4X4_KHR:
+    case COMPRESSED_RGBA_ASTC_5X4_KHR:
+    case COMPRESSED_RGBA_ASTC_5X5_KHR:
+    case COMPRESSED_RGBA_ASTC_6X5_KHR:
+    case COMPRESSED_RGBA_ASTC_6X6_KHR:
+    case COMPRESSED_RGBA_ASTC_8X5_KHR:
+    case COMPRESSED_RGBA_ASTC_8X6_KHR:
+    case COMPRESSED_RGBA_ASTC_8X8_KHR:
+    case COMPRESSED_RGBA_ASTC_10X5_KHR:
+    case COMPRESSED_RGBA_ASTC_10X6_KHR:
+    case COMPRESSED_RGBA_ASTC_10X8_KHR:
+    case COMPRESSED_RGBA_ASTC_10X10_KHR:
+    case COMPRESSED_RGBA_ASTC_12X10_KHR:
+    case COMPRESSED_RGBA_ASTC_12X12_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_4X4_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_5X4_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_5X5_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_6X5_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_6X6_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_8X5_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_8X6_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_8X8_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_10X5_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_10X6_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_10X8_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_10X10_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_12X10_KHR:
+    case COMPRESSED_SRGB8_ALPHA8_ASTC_12X12_KHR:
+      return supportedFormats.has("astc");
+
+    case COMPRESSED_R11_EAC:
+    case COMPRESSED_RG11_EAC:
+    case COMPRESSED_SIGNED_RG11_EAC:
+    case COMPRESSED_RGB8_ETC2:
+    case COMPRESSED_RGBA8_ETC2_EAC:
+    case COMPRESSED_SRGB8_ETC2:
+    case COMPRESSED_SRGB8_ALPHA8_ETC2_EAC:
+    case COMPRESSED_RGB8_PUNCHTHROUGH_ALPHA1_ETC2:
+    case COMPRESSED_SRGB8_PUNCHTHROUGH_ALPHA1_ETC2:
+      return supportedFormats.has("etc2");
+
+    case COMPRESSED_RED_RGTC1_EXT:
+    case COMPRESSED_SIGNED_RED_RGTC1_EXT:
+    case COMPRESSED_RED_GREEN_RGTC2_EXT:
+    case COMPRESSED_SIGNED_RED_GREEN_RGTC2_EXT:
+      return supportedFormats.has("rgtc");
+
+    case COMPRESSED_SRGB_S3TC_DXT1_EXT:
+    case COMPRESSED_SRGB_ALPHA_S3TC_DXT1_EXT:
+    case COMPRESSED_SRGB_ALPHA_S3TC_DXT3_EXT:
+    case COMPRESSED_SRGB_ALPHA_S3TC_DXT5_EXT:
+      return supportedFormats.has("dxt-srgb");
+    default:
+      return false;
+  }
+};
+
+type SIZE = {
```

Comment:
Caps is for constants

```suggestion
type Size = {
```

Reply:
Done.